package.json

```diff
@@ -31,5 +31,6 @@
             "_comment": "This will remove husky from when we started migrating to use prettier",
             "pre-commit": "npm uninstall husky"
         }
-    }
+    },
+    "dependencies": {}
```

Comment:
Why do we need this change?

Reply:
No idea how that ended up there. Removed it.